python/PiFinder/state.py

```diff
@@ -26,6 +27,20 @@ def __init__(self):
         self.__datetime_time = None
         self.__target = None
         self.__screen = None
+        self.__solve_pixel = config.Config().get_option("solve_pixel")
+
+    def solve_pixel(self, screen_space=False):
+        """
+        solve_pixel is (Y,X) in camera image (512x512) space
+
+        if screen_space=True, this is returned as (X,Y) in screen (128x128) space
+        """
+        if screen_space:
+            return (int(self.__solve_pixel[1] / 4), int(self.__solve_pixel[0] / 4))
```

Comment:
I've probably done this myself in some places, but at one point we'll have another screen resolution -
so maybe define this somewhere central, screen size, camera size, screen-camera ratio

Reply:
Indeed!  Work for a future (maybe not too far) day....